packages/components/src/range-control/stories/index.story.tsx

```diff
@@ -167,6 +169,7 @@ export const WithIntegerStepAndMarks: StoryFn< typeof RangeControl > =
 	MarkTemplate.bind( {} );
 
 WithIntegerStepAndMarks.args = {
+	__nextHasNoMarginBottom: true,
```

Comment:
Ah, would have been nice to have an object to declare common props. Not a blocker of course.

Reply:
Haha yes, I considered abstracting it but decided against doing it here because it would be annoying to review it in this big PR 😂